modules/db/src/test/resources/integration/template/template-config.xml

```diff
@@ -55,6 +55,11 @@
         <db:dynamic-query>${dynamicQuery}</db:dynamic-query>
     </db:template-query>
 
+    <db:template-query name="testNullParamsQuery">
+        <db:parameterized-query>SELECT * FROM PLANET WHERE POSITION = :position</db:parameterized-query>
+        <db:in-param name="position" defaultValue="null"/>
```

Comment:
I suppoused It must be a literal string value (because It's testing TemplateQuery, and It doesn't resolve MEL expressions.